core/async_job/queue.ex

```diff
@@ -54,20 +54,32 @@ defmodule AntikytheraCore.AsyncJob.Queue do
   @impl true
   defun new() :: t do
     set = :gb_sets.empty()
-    %__MODULE__{jobs: %{}, index_waiting: set, index_runnable: set, index_running: set, brokers_waiting: [], brokers_to_notify: [], abandoned_jobs: []}
+    %__MODULE__{
+      jobs:              %{},
+      index_waiting:     set,
+      index_runnable:    set,
+      index_running:     set,
+      brokers_waiting:   [],
+      brokers_to_notify: [],
+      abandoned_jobs:    [],
+    }
   end
 
   @impl true
   defun command(q1 :: v[t], cmd :: RVData.command_arg) :: {RVData.command_ret, t} do
     case cmd do
-      {{:add, job_key, job}                   , now_millis} -> insert(q1, job_key, job)                         |> maintain_invariants_and_return(now_millis)
-      {{:fetch, broker}                       , now_millis} -> fetch(q1, broker, now_millis)                    |> maintain_invariants_and_return(now_millis)
-      {{:remove_locked, job_key}              , now_millis} -> {:ok, remove_locked(q1, job_key, now_millis)}    |> maintain_invariants_and_return(now_millis)
-      {{:unlock_for_retry, job_key}           , now_millis} -> {:ok, unlock_for_retry(q1, job_key, now_millis)} |> maintain_invariants_and_return(now_millis)
-      {{:remove_broker_from_waiting_list, pid}, now_millis} -> {:ok, remove_broker(q1, pid)}                    |> maintain_invariants_and_return(now_millis)
-      {{:cancel, job_id}                      , now_millis} -> cancel_job(q1, job_id)                           |> maintain_invariants_and_return(now_millis)
-      {:get_metrics                           , now_millis} -> {metrics(q1), q1}                                |> maintain_invariants_and_return(now_millis)
-      _                                                     -> {:ok, q1} # failsafe: not to crash on unexpected command
+      {{:add, job_key, job}                   , now_millis} -> {insert(q1, job_key, job),                         now_millis}
+      {{:fetch, broker}                       , now_millis} -> {fetch(q1, broker, now_millis),                    now_millis}
+      {{:remove_locked, job_key}              , now_millis} -> {{:ok, remove_locked(q1, job_key, now_millis)},    now_millis}
+      {{:unlock_for_retry, job_key}           , now_millis} -> {{:ok, unlock_for_retry(q1, job_key, now_millis)}, now_millis}
+      {{:remove_broker_from_waiting_list, pid}, now_millis} -> {{:ok, remove_broker(q1, pid)},                    now_millis}
+      {{:cancel, job_id}                      , now_millis} -> {cancel_job(q1, job_id),                           now_millis}
+      {:get_metrics                           , now_millis} -> { {metrics(q1), q1},                               now_millis}
```

Comment:
```suggestion
      {:get_metrics                           , now_millis} -> {{metrics(q1), q1},                                now_millis}
```

Reply:
I fixed it https://github.com/access-company/antikythera/pull/51/commits/3f4bc5fb27cc82dfbf23aca3959c93948eff71d5